### ISSUE TITLE
User Journey / User stories

### ISSUE THREAD

**Issue by mkatenolan** [fac-17/Meal-planning-app#2]
### Main user journey: 
- As a user I need a way to plan weekly meals with common ingredients in order to reduce food waste.

### User Stories
- As a user I want to know straight away what the app does on the landing page.
- As a user, I want to be offered a range of recipes to choose from for inspiration.
- As a user I want to be able to select two recipes I like the look and sound of quickly and easily.
- As a user I want to be able to quickly search through the currently existing meal plans and get a summary of their common ingredients, healthiness and average time.
- As a user I want to be able to input my dietary requirements (and allergies?).
- As a user I want to be able to see my meal plan for the week at a quick glance.
- As a user I want to have access to an easily readable shopping list covering everything I need for that week.
- As a user I want to be able to send myself a meal plan and select the number of days
- As a user I want to be able to share my meal plan via social media.
- As a user I want to be able to create a new meal plan from anywhere in the app

**Comment by victormasson21** [comment]
![IMG_9384](https://user-images.githubusercontent.com/2736418/64422202-684e9700-d09b-11e9-9e3a-34d0bf9670c3.jpg)

![IMG_9399](https://user-images.githubusercontent.com/2736418/64422208-6be21e00-d09b-11e9-83e9-e070f20e0fa2.jpg)

![IMG_9401](https://user-images.githubusercontent.com/2736418/64422221-71d7ff00-d09b-11e9-8e1a-9096dd0ae32b.jpg)

**Comment by victormasson21** [comment]
### Main user journey: 
- As a user I need a way to plan weekly meals with common ingredients in order to reduce food waste.

### User Stories (Core goals)
- As a user I want to know straight away what the app does on the landing page.
- As a user, I want to be offered a range of recipes to choose from for inspiration.
- As a user I want to be able to select two recipes I like the look and sound of quickly and easily.
- As a user I want to be able to quickly search through the currently existing meal plans and get a summary of their common ingredients, healthiness and average time.
- As a user I want to be able to see my meal plan for the week at a quick glance.
- As a user I want to have access to an easily readable shopping list covering everything I need for that week.
- As a user I want to be able to send myself a meal plan and select the number of days

### User Stories (Stretch goals)

- As a user I want to be able to share my meal plan via social media.
- As a user I want to be able to create a new meal plan from anywhere in the app.
- As a user I want to be able to input my dietary requirements (and allergies?).

**Comment by victormasson21** [comment]
### User Stories - Core goal

- to know straight away what the app does on the landing page.
- to be offered a range of recipes to choose from for inspiration.
- to be able to select two recipes I like the look and sound of quickly and easily.
- to be able to quickly search through the currently existing meal plans and get a summary of their common ingredients, healthiness and average time.
- to be able to see my meal plan for the week at a quick glance.
- to see a shopping list page populated with unique items for every meal plan.
- to be able to send myself a meal plan.
- to be able to select the number of days when I create a meal plan.
- to be able to name a user plan I created only once I've selected the recipes.
- to be able to see the recipes I've already chosen
- to be able to load more recipes if I don't like any of the ones I've chosen
- to be redirected to the meal plan I've just created once it has been saved
- to see an overview of the nutrition of a recipe on its page 
- to see the required ingredients for a recipe
- to see the list of instructions for a recipe
- to see any dietry limitations on a meal plan on it's main page 
- this site to be accessible to people of all abilities
- to be see how long a recipe takes to make
- to be able to use this app on mobile

### User Stories (Stretch goals)

- As a user I want to be able to share my meal plan via social media.
- As a user I want to be able to create a new meal plan from anywhere in the app.
- As a user I want to be able to input my dietary requirements (and allergies?).
